railgun/cmd/rootcmd/env.go

```diff
@@ -22,7 +29,10 @@ func bindEnvVars(cmd *cobra.Command, _ []string) {
 		}
 
 		if envValue, envSet := os.LookupEnv(envKey); envSet {
-			cmd.Flags().Set(f.Name, envValue)
+			if err := f.Value.Set(envValue); err != nil {
```

Comment:
Can we unit test this?

Reply:
There are some unit tests that already exist for this, what do you have in mind?

Reply:
a test has been added in 9b96ca359f39861e2660750f9a8eea41bb9ce9e7 :+1: